### PR TITLE
adding the smaller backgrounds

### PR DESCRIPTION
**Pull request by AlexandreMertens**
Can you just have a last check on the cross sections before merging?

### CODE REVIEW

**Comment by OlivierBondu** [comment]
why not take the xsec from the DB ? At least if a correction is needed everyone would benefit ;)

File: plotting/plotIt/MCFiles.yml [hunk 1]
```diff
@@ -22,11 +22,46 @@
...
+'ST_t-channel_4f_leptonDecays_13TeV-amcatnlo_Fall15MiniAODv2_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 70.69
+  generated-events: 1 #984400
+  order: 5
+  group: 'ST'
+
+'ZH_HToBB_ZToLL_M125_13TeV_powheg_pythia8_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 0.86
```
Comment (OlivierBondu) [inline plotting/plotIt/MCFiles.yml]: http://cp3.irmp.ucl.ac.be/~delaere/level2/SAMADhi/index.php?-action=list&-table=dataset&-cursor=0&-skip=0&-limit=30&-mode=find&-edit=1&name=ZH&energy=13
should probably be 0.172948 (if I didn't do a mistake myself...)

File: plotting/plotIt/MCFiles.yml [hunk 2]
```diff
@@ -22,11 +22,46 @@
...
+'ST_tW_antitop_5f_inclusiveDecays_13TeV-powheg_Fall15MiniAODv2_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 35.85
+  generated-events: 1  #988500
+  order: 5
+  group: 'ST'
+
+'ST_tW_top_5f_inclusiveDecays_13TeV-powheg_Fall15MiniAODv2_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 35.85
```
Comment (OlivierBondu) [inline plotting/plotIt/MCFiles.yml]: I had 35.6 for both... from https://twiki.cern.ch/twiki/bin/view/CMS/SummaryTable1G25ns#Single_top

where are yours from ?

File: plotting/plotIt/MCFiles.yml [hunk 3]
```diff
@@ -22,11 +22,46 @@
...
+  generated-events: 1 #995600
+  order: 5
+  group: 'ST'
+
+'ST_s-channel_4f_leptonDecays_13TeV-amcatnlo_Fall15MiniAODv2_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 3.36
```
Comment (OlivierBondu) [inline plotting/plotIt/MCFiles.yml:41]: This is correct, but what I had in the DB was not ;)

File: plotting/plotIt/MCFiles.yml (hunk 2 above)
Comment (AlexandreMertens) [inline plotting/plotIt/MCFiles.yml]: you're right! thanks for the checks!

**Pull request merged**